src/locale.js

```diff
@@ -76,25 +76,69 @@ export default class Locale {
 		this.contentLanguageDirection = getLanguageDirection( this.contentLanguage );
 
 		/**
-		 * Translates the given string to the {@link #uiLanguage}. This method is also available in
+		 * Translates the given string to the {@link #uiLanguage}.
+		 *
+		 * This method is also available in
 		 * {@link module:core/editor/editor~Editor#t} and {@link module:ui/view~View#t}.
 		 *
-		 * The strings may contain placeholders (`%<index>`) for values which are passed as the second argument.
+		 * The string may contain placeholders (`%<index>`) for given dynamic values.
 		 * `<index>` is the index in the `values` array.
 		 *
-		 *		editor.t( 'Created file "%0" in %1ms.', [ fileName, timeTaken ] );
-		 *
-		 * This method's context is statically bound to Locale instance,
-		 * so it can be called as a function:
-		 *
-		 *		const t = this.t;
-		 *		t( 'Label' );
+		 *		editor.t( 'Created file "%0" in %1ms', [ fileName, time ] );
 		 *
 		 * @method #t
-		 * @param {String} str The string to translate.
+		 * @param {String} msgid The string to translate.
 		 * @param {String[]} [values] Values that should be used to interpolate the string.
 		 */
-		this.t = ( ...args ) => this._t( ...args );
+		this.t = ( msgid, values ) => this._t( msgid, values );
+
+		/**
+		 * Translates the given string to the {@link #uiLanguage}.
+		 *
+		 * This method allows passing the context as the first argument.
+		 * A context is used to provide a description or example of how the translated string will be used.
+		 * An unique context makes the whole translation unique and should be presented to the translator.
+		 *
+		 * The string may contain placeholders (`%<index>`) for given dynamic values.
+		 * `<index>` is the index in the `values` array.
+		 *
+		 *		editor.ct( 'Created file "x"', 'file "%0"', [ fileName ] );
+		 *
+		 * If at least one of the placeholder values is not a string or number,
+		 * then an array of all the literal and dynamic parts will be returned
```

Comment:
. missing at the end of the sentence.

Please expand with an example. It was difficult for me to understand what this means and how I could use it.